.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+sudo: required
```

Comment:
I would prefer a container based env based on 14.04 but apparently there is no such thing? It would be nice to know if the container based or the 14.04 infrastructure is supposed to be the future?